### PR TITLE
838 fix issue with metadata version check in metadata portal autosigner

### PR DESCRIPTION
**Pull request by rlaferla**
Closes #838 fix issue with metadata version check in metadata portal autosigner

### CODE REVIEW

**Review by johnmela — APPROVED** [review]

File: cli/src/autosigner/mod.rs [hunk 1]
```diff
@@ -123,18 +127,33 @@ pub(crate) async fn autosign_from_github(config: AppConfig) -> anyhow::Result<()
...
+            for metadata_file_version in map.keys() {
+                if &wasm.version > metadata_file_version {
+                    needs_updating = true;
+                    info!(
+                        "The GitHub version {} is newer than {} in the file.",
+                        &wasm.version, metadata_file_version
+                    );
+                    break;
+                } else {
+                    continue;
```
Comment (enddynayn) [inline cli/src/autosigner/mod.rs:143]: If we never reach that condition, does it automatically continue?

File: cli/src/autosigner/mod.rs [hunk 2]
```diff
@@ -123,18 +127,33 @@ pub(crate) async fn autosign_from_github(config: AppConfig) -> anyhow::Result<()
...
+                }
+            }
+            if needs_updating == false {
                 info!("🎉 {} is up to date!", chain.name);
                 continue;
```
Comment (enddynayn) [inline cli/src/autosigner/mod.rs:148]: same here?

File: cli/src/autosigner/mod.rs (hunk 1 above)
Comment (rlaferla) [reply cli/src/autosigner/mod.rs:143]: Good catch 🎣!

**Review by enddynayn — APPROVED** [review]
👍 Nice!

File: cli/src/autosigner/mod.rs (hunk 2 above)
Comment (rlaferla) [reply cli/src/autosigner/mod.rs:148]: No, because this is not the inner "for" loop but the "outer" for loop.

Comment (enddynayn) [reply cli/src/autosigner/mod.rs:148]: Right but we are continuing whether or not needs_updating == false. I think it's the last condition in the outer loop.

**Pull request merged**